config/test.js

```diff
@@ -6,5 +6,6 @@ module.exports = {
   DB_HOST: 'localhost',
   DB_PORT: 5432,
   DB_PASSWORD: '',
-  DB_USER: 'sfmovies_user'
+  DB_USER: 'sfmovies_user',
+  DB_DEBUG: true
```

Comment:
You don't have to make any changes right now, but if you ever get tired of seeing the bookshelf output in the tests, you can also have this variable come from the environment so that you can decide on a test-run-by-test-run basis if you want the debug output. In other words...

```js
DB_DEBUG: process.env.DB_DEBUG
```

```sh
$ DB_DEBUG=true npm test # shows db debug info
$ npm test               # does not show db debug info
```